From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 6 Dec 2025 21:10:34 +0800
Subject: disable use of ppc64el assembly code

Bug-Debian: https://bugs.debian.org/1092890
Last-Update: 2025-02-02
---
 meson.build | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/meson.build b/meson.build
index 095aec5..38292d8 100644
--- a/meson.build
+++ b/meson.build
@@ -1356,11 +1356,6 @@ elif host_machine.cpu_family() == 'sparc64'
     with_asm_arch = 'sparc'
     pre_args += ['-DUSE_SPARC_ASM']
   endif
-elif host_machine.cpu_family() == 'ppc64' and host_machine.endian() == 'little'
-  if system_has_kms_drm
-    with_asm_arch = 'ppc64le'
-    pre_args += ['-DUSE_PPC64LE_ASM']
-  endif
 elif host_machine.cpu_family() == 'mips64' and host_machine.endian() == 'little'
   if system_has_kms_drm
     with_asm_arch = 'mips64el'
